gen_cert.py

```diff
@@ -149,6 +151,7 @@ def interstitial_factory():
                 for key, value in cert_data.get('interstitial', {}).items()
             }
             self.interstitial_texts.update(interstitial_dict)
+            self.timezone = cert_data.get('timezone', settings.DEFAULT_TIMEZONE)
```

Comment:
A side benefit is that this improves the chance of matching a regex if you're searching for the _word_ `<timezone>` instead of the _substring_ `*timezone*`.